tcp_queue_length/manifest.json

```
@@ -0,0 +1,27 @@
+{
+  "maintainer": "help@datadoghq.com",
+  "manifest_version": "0.0.1",
+  "metric_prefix": "tcp_queue.",
+  "metric_to_check": "tcp_queue.rqueue.size",
+  "name": "tcp_queue_length",
+  "short_description": "Track the size of the TCP buffers with Datadog",
+  "guid": "0468b098-43bd-4157-8a01-14065cfdcb7b",
+  "support": "core",
+  "supported_os": [
+    "linux"
+  ],
+  "public_title": "Datadog-TCP queue length Integration",
+  "categories": [
+    "network"
+  ],
+  "type": "check",
+  "is_public": true,
+  "display_name": "TCP queue length",
+  "creates_events": true,
+  "integration_id": "tcp_queue_length",
+  "assets": {
+    "monitors": {},
+    "dashboards": {},
+    "service_checks": {}
```

We also need to include service checks under an `assets/` directory. If the tcp_queue_length check does not emit any service checks, the json file can just be `[]`.

```suggestion
        "service_checks": "assets/service_checks.json"

```

CI is failing because of missing file.

---

Unfortunately, `tcp_queue_length` doesn’t provide any service check yet.
I’ll need to add one in the corresponding go core check.